BEIMA.Backend.Test/MongoConnectorTest.cs

```diff
@@ -0,0 +1,227 @@
+﻿using NUnit.Framework;
+using BEIMA.Backend.MongoService;
+using System.IO;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System;
+using MongoDB.Bson;
+
+namespace BEIMA.Backend.Test
+{
+    [TestFixture]
+    public class MongoConnectorTest
+    {
+        //private MongoConnector? mongo;
+        class LocalSettings
+        {
+            public bool IsEncrypted { get; set; }
+            public Dictionary<string, string> Values { get; set; }
+        }
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            //For local testing, the try block will be used to instantiate local variables.
+            //For cloud testing, this will throw an exception, and it will use the env variables
+            //defined in beima.yml.
+            try
+            {
+                string[] paths = { "..", "..", "..", "..", "BEIMA.Backend", "local.settings.json" };
+                string combinedPath = Path.Combine(paths);
+                string fullPath = Path.GetFullPath(combinedPath);
+                var settings = JsonConvert.DeserializeObject<LocalSettings>(
+                    File.ReadAllText(fullPath));
+
+                foreach (var setting in settings.Values)
+                {
+                    Environment.SetEnvironmentVariable(setting.Key, setting.Value);
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.ToString());
+            }
+            
+        }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+        }
+
+        [Test]
+        public void InitialState_Action_ExpectedResult()
```

Comment:
I think we can remove this, unless we want to leave it as an example

Reply:
I think I will leave this in, it is easy to see the format and this test is a guaranteed pass. If it fails, then we know something is wrong with the framework.

Reply:
I think we can remove it. If this is a valuable example, we should document this in a Testing Guide/Coding Standards that we can create and use as reference. If something is wrong with the framework, I think the `dotnet test` command will catch it before it runs any tests. Or we make a separate test the verifies the framework is working. It adds unneeded coupling to this test file.

If we agree to keep it, we should add a comment stating why. If a new developer were to come in and see this test case, it wouldn't be very clear as to why we left it. Also, we should remove the OneTimeTearDown, Setup, and TearDown methods since they are doing nothing. Unless they are needed to run the test in the framework, then we should add a comment indicating that.

Reply:
I think that we should remove it as its just unnecessary code. I agree with Ashlyn that we could put it in a document if we want to keep it around.

Reply:
Fixed [6f34fb6](https://github.com/cs481-ekh/s22-beima/pull/67/commits/6f34fb69b240040dce07b88ac263f3df03d88033)